javascript/fyrejet-uwebsockets/app.js

```diff
@@ -6,17 +6,18 @@ var app = fyrejet({
 });
 
 app.set("etag", false);
+app.set('x-powered-by', false)
 
 app.get("/", function (req, res) {
-  res.send("");
+  res.sendLite("");
```

Comment:
is this intended to be used in **production** ?

I mean, what are the differences between `send` and `sendLite` ?

Reply:
Yes. Fyrejet is heavily based on restana with res.sendLite being the renamed restana’s res.send. 

The goal of the framework, however, is to provide Express-like API with significantly improved performance. So it also implements most of Express API, besides a few deprecated features.

 The default res.send is thus a highly modified function of the same name from express.js, which, in comparison to res.sendLite also provides some features related to ETag, caching, etc. 

TL;DR: difference between res.sendLite and res.send is that the former has less features, but is much faster